devito/dse/rewriters.py

```diff
@@ -408,5 +409,26 @@ def _extract_sum_of_products(self, cluster, template, **kwargs):
 
 class CustomRewriter(AggressiveRewriter):
 
+    passes_mapper = {
+        'extract_sum_of_products': AggressiveRewriter._extract_sum_of_products,
+        'factorization': AggressiveRewriter._factorize,
+        'eliminate_inter_stencil_redundancies':
```

Comment:
some passes are missing

Reply:
Added from all Rewriters now. Is that something we want ? I guess yes?

Reply:
I think we need super short names for these passes. I'll try listing them, in order, but please feel free to improve them

```
extract_sop
factorize
gcse  # Generalized Common Sub-expressions Elimination  (this is new terminology I'm introducing)
cse  # Common Sub-expressions Elimination 
extract_invariants
extract_indices
opt_trascendentals
```